src/PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp

```diff
@@ -24,6 +24,7 @@ template <bool IsRelativistic>
 class IdealFluid;
 template <bool IsRelativistic>
 class PolytropicFluid;
+class Spectral;
```

Comment:
This isn't used, but I think it should be.

Reply:
Turns out it is unnecessary, as the Spectral EoS is not a templated class (as opposed to all other EoS...).

Reply:
Doesn't it need to be added to one of the `DerivedClasses` lists?